src/Application/Application.cpp

```
@@ -90,18 +90,40 @@ void tryOpenConsole() {
 }
 
 
+void checkVulkanLoader() {
+    SystemUtils::VkLoaderDiag vkLoaderDiag = SystemUtils::VulkanLoaderExists();
+    std::string vkLoader = VULKAN_LOADER;
+    std::string vkLoaderErrMsg;
+
+    switch (vkLoaderDiag.errType) {
+    case SystemUtils::VkLoaderDiag::CANNOT_BE_LOCATED:
```

Clang (14.0.6)

[warning] switch has 2 consecutive identical branches

(bugprone-branch-clone)